metadata.yaml

```
@@ -29,13 +32,7 @@ requires:
     interface: s3
     limit: 1
 
-resources:
-  patroni:
-    type: file
-    filename: patroni.tar.gz
-    description: Patroni python package.
-
 storage:
   pgdata:
     type: filesystem
-    location: /var/lib/postgresql/data
+    location: /var/snap/charmed-postgresql/common/postgresql
```

data of all components are important
```suggestion
    location: /var/snap/charmed-postgresql/common
```